src/sonic-bgpcfgd/tests/data/internal/peer-group.conf/result_chasiss_packet.conf

```diff
@@ -3,15 +3,15 @@
 !
   neighbor INTERNAL_PEER_V4 peer-group
   neighbor INTERNAL_PEER_V6 peer-group
-  address-family ipv4
     neighbor INTERNAL_PEER_V4 update-source Loopback4096 
```

Comment:
should we remove this indentation if neighbor update-source is not in address-family group.

Reply:
updated.